internal/secret/secrettype/secret.go

```diff
@@ -266,40 +267,40 @@ var DefaultRules = map[string]Meta{
 	},
 	Oracle: {
 		Fields: []FieldMeta{
-			{Name: OracleUserOCID, Required: true, Description: "Your Oracle user OCID. Find more about, generating public key and fingerprint here: https://banzaicloud.com/docs/pipeline/secrets/providers/oci_auth_credentials/"},
-			{Name: OracleTenancyOCID, Required: true, Description: "Your tenancy OCID"},
-			{Name: OracleAPIKey, Required: true, Description: "Your public key"},
-			{Name: OracleAPIKeyFingerprint, Required: true, Description: "Fingerprint of you public key"},
-			{Name: OracleRegion, Required: true, Description: "Oracle region"},
-			{Name: OracleCompartmentOCID, Required: true, Description: "Your compartment OCID"},
+			{Name: OracleUserOCID, Required: true, IsSafeToDisplay: true, Description: "Your Oracle user OCID. Find more about, generating public key and fingerprint here: https://banzaicloud.com/docs/pipeline/secrets/providers/oci_auth_credentials/"},
+			{Name: OracleTenancyOCID, Required: true, IsSafeToDisplay: true, Description: "Your tenancy OCID"},
+			{Name: OracleAPIKey, Required: true, IsSafeToDisplay: true, Description: "Your public key"},
```

Comment:
**Question**: are we sure this is a public key?
API keys are usually private that's why I'm asking, the name and the description are somewhat contradicting each other according to my understanding, but that's just my intuition.

Same [at](https://github.com/banzaicloud/pipeline/pull/3202/files#diff-3aae6f1133ed7ec044650bb3eb2efec4R44) `internal/secret/types/type_oracle.go:44`.

Reply:
Yes, this is a public key (https://banzaicloud.com/docs/pipeline/secrets/providers/oci_auth_credentials/#generate-api-token). In Oracle side we need to upload a public key to use this secret

Reply:
Cool, it's all good then.